tests/FileHelperTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+declare(strict_types = 1);
+
+namespace Spaze\PHPStan\Rules\Disallowed;
+
+use Generator;
+use PHPUnit\Framework\TestCase;
+use PHPStan\File\FileHelper as PHPStanFileHelper;
```

Comment:
This seems as an unused `use` but if you'll go the "pass it as a param" way it might suddenly be needed.